src/Symfony/Component/HttpKernel/Fragment/InlineFragmentRenderer.php

```diff
@@ -59,6 +59,11 @@ public function render($uri, Request $request, array $options = array())
             // want that as we want to preserve objects (so we manually set Request attributes
             // below instead)
             $attributes = $reference->attributes;
+            if (isset($options['is_fallback']) && $options['is_fallback'] )
+            {
```

Comment:
Not sure why fabbot did not catch it, but we put opening braces on the same line as the `if`